package.json

```diff
@@ -7,7 +7,7 @@
     "create-edx-react-app": "build/index.js"
   },
   "scripts": {
-    "build": "rm -rf build/ && babel src/ -d build/ --ignore 'src/templates' --copy-files --include-dotfiles",
+    "build": "rm -rf build/ && babel src/ -d build/ --copy-files --include-dotfiles",
```

Comment:
[The `--delete-dir-on-start` option](https://github.com/babel/babel/pull/6187) could be used instead of `rm -rf`